wire-ios-sync-engine/Tests/Source/Integration/APNSTests+Swift.swift

```diff
@@ -18,10 +18,11 @@
 
 import Foundation
 import WireMockTransport
+import XCTest
 
 class APNSTests_Swift: APNSTestsBase {
 
-    func testThatItUpdatesApplicationBadgeCount_WhenReceivingATextMessage() {
+    func diabled_testThatItUpdatesApplicationBadgeCount_WhenReceivingATextMessage() {
```

Comment:
we disable this one? @typfel ?

Reply:
Oh might be me debugging something, try enabling it again.